src/SprykerEco/Zed/Heidelpay/Business/HeidelpayFacade.php

```diff
@@ -75,6 +92,36 @@ public function authorizePayment(OrderTransfer $orderTransfer): void
             ->authorize($orderTransfer);
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     *
+     */
+    public function authorizeOnRegistrationPayment(OrderTransfer $orderTransfer)
```

Comment:
please check return types in this facade and it its interface

Reply:
Skipped (@alex-galych ).